packages/wpcom-proxy-request/src/index.js

```diff
@@ -112,8 +112,8 @@ const request = ( originalParams, fn ) => {
 		install();
 	}
 
-	// generate a uid for this API request
-	const id = uid();
+	// generate a uuid for this API request
+	const id = uuidv4();
```

Comment:
I'm sad we can't use `Symbol()` because I don't believe that would survive the message channel 😭

Reply:
Yeah, was wondering if we could replace the `uuid` stuff with something like that. Not sure what would happen to a `Symbol` that's `postMessage()`d, I'd need to check. (I wanted to keep the changes here rather contained for now :sweat_smile: )

Reply:
```js
window.onmessage = console.log
window.postMessage({callback: 'unique-id'},'*') // 👍
window.postMessage({callback: Symbol()},'*') // 💥 
VM2252:1 Uncaught DOMException: Failed to execute 'postMessage' on 'Window': Symbol() could not be cloned.
```